### ISSUE TITLE
ISLE Repo size is too large?

### ISSUE THREAD

**Issue by g7morris** [Islandora-Collaboration-Group/ISLE#210]
#### Issue description
The size of the ISLE git repository and the speed of cloning seem just off. It just takes quit awhile to clone this directory.

I suspect this may be due to the use of image submodules, the vagrant, ansible directories etc. I am also not convinced that our previously failed attempt to use `Gitfs` didn't quite get cleaned up? 

#### For an issue, describe steps to reproduce the issue

System setup (OS information, software versions, etc):

Steps:
1. `cd /tmp`
2. `time git clone https://github.com/Islandora-Collaboration-Group/ISLE.git`
3. results for a directory that is `177M`  were
```
real	0m29.424s
user	0m8.052s
sys	0m2.660s
```

##### What's the expected result?

- That the ISLE git repository is small < 10 MB
- That the ISLE git repository is quick to clone

##### What's the actual result?

- That the ISLE git repository is > 177MB
- That the ISLE git repository is slow to clone (30+ seconds depending on Internet connection)

**Comment by noahwsmith** [comment]
Confirming:

```
Noahs-MacBook-Pro:tmp noah$ time git clone https://github.com/Islandora-Collaboration-Group/ISLE.git
Cloning into 'ISLE'...
remote: Enumerating objects: 4, done.
remote: Counting objects: 100% (4/4), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 15383 (delta 0), reused 2 (delta 0), pack-reused 15379
Receiving objects: 100% (15383/15383), 173.61 MiB | 12.86 MiB/s, done.
Resolving deltas: 100% (4251/4251), done.

real	0m14.255s
user	0m5.929s
sys	0m1.984s
```

**Comment by g7morris** [comment]
This was fixed in release 1.4 with the git cleanup. Closing ticket.
